AlphaWallet/Tokens/Types/TokenObject.swift

```diff
@@ -47,7 +47,7 @@ class TokenObject: Object {
     }
 
     var address: Address {
-        return Address(string: contract)!
+        return Address(uncheckedAgainstNullAddress: Constants.nullAddress)!
```

Comment:
You'll probably catch it, but just in case I forgot:

`contract`

not:

`Constants.nullAddress`

Reply:
Woops, I thought I put that!